app/src/androidTest/java/org/kiwix/kiwixmobile/main/TopLevelDestinationTest.kt

```diff
@@ -46,11 +53,7 @@ class TopLevelDestinationTest : BaseActivityTest() {
         clickFileTransferIcon {
         }
       }
-      clickDownloadOnBottomNav {
-        assertLibraryListDisplayed()
-        clickOnGlobeIcon {
-        }
-      }
+      clickDownloadOnBottomNav(OnlineLibraryRobot::assertLibraryListDisplayed)
```

Comment:
We shouldn't delete the locale language test cases.

Reply:
hi @gouri-panda ,

we are not deleting this test we are running this test separately https://github.com/kiwix/kiwix-android/blob/develop/app/src/androidTest/java/org/kiwix/kiwixmobile/language/LanguageFragmentTest.kt here.